packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -496,177 +496,187 @@ export default class TableResize implements EditorPlugin {
                                     ? parseFloat(
-                                          cell.td.style.width.substr(
+                                          cell.td.style.height.substr(
                                               0,
-                                              cell.td.style.width.length - 2
+                                              cell.td.style.height.length - 2
                                           )
                                       )
-                                    : cell.td.getBoundingClientRect().right -
-                                      cell.td.getBoundingClientRect().left;
-                                const newWidth = originalWidth * ratioX;
-                                cell.td.style.boxSizing = 'border-box';
-                                if (newWidth >= MIN_CELL_WIDTH) {
-                                    cell.td.style.wordBreak = 'break-word';
-                                    cell.td.style.width = `${newWidth}px`;
-                                }
-                            }
-
-                            if (shouldResizeY) {
-                                if (j == 0) {
-                                    const originalHeight =
-                                        cell.td.getBoundingClientRect().bottom -
-                                        cell.td.getBoundingClientRect().top;
-                                    const newHeight = originalHeight * ratioY;
-                                    if (newHeight >= MIN_CELL_HEIGHT) {
-                                        cell.td.style.height = `${newHeight}px`;
-                                    }
-                                } else {
-                                    cell.td.style.height = '';
+                                    : cell.td.getBoundingClientRect().bottom -
+                                      cell.td.getBoundingClientRect().top;
+                                const newHeight = originalHeight * ratioY;
+                                if (newHeight >= MIN_CELL_HEIGHT) {
+                                    cell.td.style.height = `${newHeight}px`;
                                 }
+                            } else {
+                                cell.td.style.height = null;
                             }
                         }
                     }
                 }
             }
-            rect = normalizeRect(this.currentTable.getBoundingClientRect());
-            currentBorder = this.isRTL ? rect.left : rect.right;
-            this.currentTable.setAttribute(
-                this.isRTL ? 'currentLeftBorder' : 'currentRightBorder',
-                currentBorder.toString()
-            );
+        }
+        rect = normalizeRect(vtable.table.getBoundingClientRect());
+        currentBorder = this.isRTL ? rect.left : rect.right;
+        vtable.table.setAttribute(
+            this.isRTL ? 'currentLeftBorder' : 'currentRightBorder',
```

Comment:
again, why need to remember this value?

Reply:
the table size is being changed during resizing so we need to remember this original value

Reply:
Ok, I agree we need to remember something during resize. So my next question is, is it the only solution to remember it in DOM? Can we remember it somewhere else, such as local var, or in VTable?

Reply:
Fixed..Interesting that you are able to see my comment before submitting the review...